test-fixture/src/lib.rs

```diff
@@ -187,6 +193,7 @@ where
     )
     .expect("create a client");
 
+    #[cfg(feature = "qlog")]
```

Comment:
I wonder if you can also avoid having to do this.  The extra work here is pretty trivial if calling `set_qlog` ultimately hits a noop.  It's not like this is production code.